keanu-python/tests/test_tensor.py

```diff
@@ -75,7 +71,7 @@ def test_cannot_pass_generic_to_Tensor(generic) -> None:
 def test_ndarray_passed_to_Tensor_creates_nonscalar_tensor(arr: primitive_types, expected_java_class: str) -> None:
     ndarray = np.array(arr)
     t = Tensor(ndarray)
-    assert_java_class(t, expected_java_class)
```

Comment:
seems a shame to delete this... can't you still assert that it inherits from `DoubleTensor`?